src/features/commands/poll.js

```diff
@@ -169,7 +169,7 @@ module.exports = {
                             )
                             .setImage(quickChartClient.getUrl());
 
-                        interaction.editReply({ embeds: [resultsEmbed] });
+                        interaction.editReply({ embeds: [resultsEmbed], ephemeral: true });
```

Comment:
This one is fine as it is, it should be set for case "list": and it is set there, it just doesn't work properly due to message being deferred, workaround would be set it to reply instead of defferreply and I believe that ephemeral switches would work then.

Reply:
Yeah